### PR TITLE
finished the side navigation

### PR DESCRIPTION
**Pull request by AhmadKawasmi**

### CODE REVIEW

File: src/components/pages/Dashboard.jsx [hunk 1]
```diff
@@ -3,15 +3,19 @@ import { useNavigate } from "react-router-dom";
...
 import { FaTrash, FaClipboardList, FaPlus } from "react-icons/fa";
 import "../style/Dashboard.css";
 
-const Dashboard = ({ user }) => {
+const Dashboard = ({ user , handleOpenNav}) => {
   const [forms, setForms] = useState([]);
   const navigate = useNavigate();
 
+  const ahmad = function(){
+    handleOpenNav()
```
Comment (rami-monday) [inline src/components/pages/Dashboard.jsx]: Ahmad???? You don't even need this

File: src/components/subComponents/SideNavigation.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import SignOut from './SignOut'
+import '../style/SideNavigation.css'
+import { MdClear , MdSpaceDashboard , MdEmail} from "react-icons/md";
+import { FaAlignJustify , FaHome } from "react-icons/fa"
+import { FaInfoCircle } from "react-icons/fa"
+
+const handleCloseNav = function(){
+    document.querySelector(".sideNavigation").style.marginLeft = "-80%";
+    document.querySelector(".openSideNav").style.display = "block"
+    document.querySelector(".closeSideNav").style.display = "none"
```
Comment (rami-monday) [inline src/components/subComponents/SideNavigation.jsx]: Whyy? React is a powerful tool, I'll show you today how to do this with react

File: src/components/subComponents/Header.jsx [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
+export const Header = ({ ahmad }) => {
     const navigator = useNavigate();
     const entranceToSignUp = function () {
         navigator("/signUp");
       };
+
+    const handleNavigation = function(){
+      ahmad()
+    }
+
```
Comment (rami-monday) [inline src/components/subComponents/Header.jsx]: No need to create another function here

File: src/components/pages/Responses.jsx [hunk 4]
```diff
@@ -1,75 +1,75 @@
...
-// import { CSVLink } from "react-csv";
-import { DownloadTableExcel } from "react-export-table-to-excel";
-import "../style/Responses.css";
+// import React, { useEffect, useState, useRef } from "react";
+// import { useParams } from "react-router-dom";
+// import { getOneFormById } from "../../services/form";
+// import { getResponses } from "../../services/responses";
+// // import { CSVLink } from "react-csv";
+// import { DownloadTableExcel } from "react-export-table-to-excel";
+// import "../style/Responses.css";
```
Comment (rami-monday) [inline src/components/pages/Responses.jsx:11]: What happened to `Responses` ?

**Review by rami-monday — CHANGES_REQUESTED** [review]

File: src/components/pages/Responses.jsx (hunk 4 above)
Comment (AhmadKawasmi) [reply src/components/pages/Responses.jsx:11]: we dont need any responses

**Review by rami-monday — APPROVED** [review]

**Pull request merged**